src/modules/mc_hover_thrust_estimator/hover_thrust_estimator_params.c

```diff
@@ -49,7 +49,7 @@
  * @decimal 4
  * @min 0.0001
  * @max 1.0
- * @unit normalized_thrust/s
+ * @unit %alized_thrust/s
```

Comment:
this is not that clear

Reply:
That is a search replace error. Will Fix. Thanks @BazookaJoe1900 

@bkueng FYI, I am reverting back to `normalized_thrust/s`. Is there a better unit?

Reply:
Hmm let's ask @bresch what he wants as he's the author of all of these params.

Reply:
We could use %/s but then we need to scale the result by 100.
If we can keep `normalized_thrust/s` I'm happy with it.